### ISSUE TITLE
[FEAT]: Author level filter for monitored

### ISSUE THREAD

**Issue by WallaWallaWA** [Readarr/Readarr#1215]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

No problem, just an enhancement to the UI.

### Describe the solution you'd like

Problem: For authors with dozens or hundreds of books, including compilations, collections, etc., where only a subset or a single series of books is monitored, it makes it difficult to scroll through to try and see what is still left to search for or upgrade.

Feature Request: At the Author level, a filter for monitored/unmonitored books, same as at Library view.

Benefit: By limiting the view to just monitored, it provides a cleaner, faster display of what is still left to search for. 

### Describe alternatives you've considered

Not aware of any other way to do this with the current UI

### Anything else?

Same as the Filter on the Library page, except at the Author level

[AB#1498](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/1498)

**Comment by bakerboy448** [comment]
I know it's not at the author view, but the `book-index` (testing) branch can also accomplish what you want with the new book view
![image](https://user-images.githubusercontent.com/55419169/130710119-e4056e4f-f2b4-435d-8b77-bbe8d1bf549d.png)

ref: https://github.com/Readarr/Readarr/pull/1152

**Comment by WallaWallaWA** [comment]
Thanks. On Windows, nightly (0.1.0.944) so don't have  that view available. Did try with custom filters in Bookshelf view but it's still hard to slog through.

**Comment by austinwbest** [comment]
It'll get merged in the next day or two. Build from source or docker would be your only way for PR branches. I'd suggest waiting though

**Comment by WallaWallaWA** [comment]
Yeah, definitely will wait...not an urgent issue, just nice to have - thanks!
